infra/pr_helper.py

```diff
@@ -223,6 +228,26 @@ def get_pull_request_number(self, commit):
       return None
     return pr_response.json()[0]['number']
 
+  def get_past_contributors(self, project_path):
```

Comment:
It's still valuable to get the entire list, but we should just annotate the returned contributors with whether or not they're verified. e.g.

```
class Contributor:
  username: str
  email: str
  verified: bool
```

Then when we render them in the comment, we just add "(unverified)" when a username is not verified.

Reply:
Ah actually,  I see you're actually already doing most of this.

Since we want to make "unverified" more prominent, can you make it show that instead of "(verified)" ?